third-party/llvm/Makefile

```diff
@@ -36,6 +36,7 @@ $(LLVM_FILE):
 	cd $(LLVM_BUILD_DIR) && $(MAKE)
 	cd $(LLVM_BUILD_DIR) && $(MAKE) install
 	cd $(LLVM_BUILD_DIR) && cd test && $(MAKE) lit.site.cfg
+	../../util/config/gather-clang-sysroot-arguments > configured-clang-sysroot-arguments
```

Comment:
If I understand this correctly, configured-clang-sysroot-arguments is placed under third-party/llvm/ dir. I think it could go under the LLVM_BUILD_DIR, or it needs to be added to the top level .gitignore and to the cleanall and clobber targets in this makefile (possibly clean target, too).